chorus_lib/src/transport/http.rs

```diff
@@ -101,16 +101,14 @@ impl Transport for HttpTransport {
 
     fn send<V: Portable>(&self, from: &str, to: &str, data: &V) -> () {
         let (hostname, port) = self.config.get(to).unwrap();
-        retry(
-            Exponential::from_millis(10).map(jitter).take(10),
```

Comment:
1. exponential retry is not suitable because the interval gets too long
2. it should keep retrying forever